app/models/google_calendar/event_finder.rb

```diff
@@ -0,0 +1,89 @@
+module GoogleCalendar
+  class EventFinder
+    GOOGLE_EVENT_DECLINED_RESPONSE = 'declined'.freeze
+    LISTING_FIELDS = 'items(id, start, end, summary, recurrence, creator, attendees(self, responseStatus))'.freeze
+
+    def initialize(credentials, user_email)
+      @credentials = credentials
+      @user_email = user_email
+      @calendar_service = GoogleCalendar::Client.new(credentials).calendar_service
+    end
+
+    def list_events(starting, ending)
+      all_events = daily_events_container
```

Comment:
you can extract this into separate method 
```
```
def all_events
@all_events = Hash[(1..CalendarHelper::WEEK_LENGTH).map { |i| [i, []] }]
end
```
so than you don't have to use pass it as method variable